modin/pandas/base.py

```diff
@@ -3191,7 +3227,7 @@ def apply(self, func, *args, **kwargs):
 
         dataframe = DataFrame(
             query_compiler=query_comp_op(
-                self.resample_args,
+                list(self.resample_args.values()),
```

Comment:
```suggestion
                self.resample_args.values(),
```

Here and below the list construction is superfluous.

Reply:
This construction list was deleted.